_shared/hooks/post_gen_project.py

```diff
@@ -19,6 +19,10 @@ def remove_conditional_files():
     paths_to_remove.extend(["package.json", "yarn.lock"])
     {% endif %}
 
+    {% if cookiecutter.get("db") != "yes" %}
+    paths_to_remove.extend(["bin/create-db"])
+    {% endif %}
```

Comment:
Don't create the `bin/create-db` file if the `db` option isn't enabled in `cookiecutter.json`.